runtime/doc/options.txt

```diff
@@ -824,7 +824,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	again not rename the file.
 
 						*'backupdir'* *'bdir'*
-'backupdir' 'bdir'	string	(default ".,$XDG_DATA_HOME/nvim/backup//")
+'backupdir' 'bdir'	string	(default "$XDG_STATE_HOME/nvim/backup//")
```

Comment:
The `.,` prefix should remain. https://github.com/neovim/neovim/pull/15433#issuecomment-907200804